app/services/XmlBuilderService.scala

```diff
@@ -53,99 +52,105 @@ class XmlBuilderService {
+      meta.firstAndLastTransfer,
+      "FirAndLasTraMES23")
+
+  private def buildHeaderNode(header: Header, arrivalNotificationDate: String): NodeSeq = <HEAHEA>
+    <DocNumHEA5>
+      {header.movementReferenceNumber}
+    </DocNumHEA5>{buildOptionalElem(header.customsSubPlace, "CusSubPlaHEA66")}<ArrNotPlaHEA60>
+      {header.arrivalNotificationPlace}
+    </ArrNotPlaHEA60> <ArrNotPlaHEA60LNG>
+      {header.languageCode}
+    </ArrNotPlaHEA60LNG>{buildOptionalElem(header.arrivalAgreedLocationOfGoods, "ArrAgrLocCodHEA62") ++ buildOptionalElem(header.arrivalAgreedLocationOfGoods, "ArrAgrLocOfGooHEA63")}<ArrAgrLocOfGooHEA63LNG>
+      {header.languageCode}
+    </ArrAgrLocOfGooHEA63LNG>{buildOptionalElem(header.arrivalAgreedLocationOfGoods, "ArrAutLocOfGooHEA65")}<SimProFlaHEA132>
+      {header.simplifiedProcedureFlag}
+    </SimProFlaHEA132> <ArrNotDatHEA141>
+      {arrivalNotificationDate}
+    </ArrNotDatHEA141>
+  </HEAHEA>
+
+  private def buildTraderDestinationNode(traderDestination: TraderDestination): NodeSeq = <TRADESTRD>
+    {buildOptionalElem(traderDestination.name, "NamTRD7") ++ buildOptionalElem(traderDestination.streetAndNumber, "StrAndNumTRD22") ++ buildOptionalElem(traderDestination.postCode, "PosCodTRD23") ++ buildOptionalElem(traderDestination.city, "CitTRD24") ++ buildOptionalElem(traderDestination.countryCode, "CouTRD25")}<NADLNGRD>
+      {traderDestination.languageCode}
+    </NADLNGRD>{buildOptionalElem(traderDestination.eori, "TINTRD59")}
+  </TRADESTRD>
+
+  private def buildOfficeOfPresentationNode(customsOfficeOfPresentation: CustomsOfficeOfPresentation): NodeSeq = <CUSOFFPREOFFRES>
+    <RefNumRES1>
+      {customsOfficeOfPresentation.presentationOffice}
+    </RefNumRES1>
+  </CUSOFFPREOFFRES>
 
   private def buildEnRouteEventsNode(arrivalNotificationRequest: ArrivalNotificationRequest): NodeSeq = arrivalNotificationRequest.enRouteEvents match {
-
     case None => NodeSeq.Empty
     case Some(enRouteEvent) =>
       enRouteEvent.map {
         event =>
           <ENROUEVETEV>
-            <PlaTEV10>{event.place}</PlaTEV10>
-            <PlaTEV10LNG>{arrivalNotificationRequest.header.languageCode}</PlaTEV10LNG>
-            <CouTEV13>{event.countryCode}</CouTEV13>
-            <CTLCTL>
-              <AlrInNCTCTL29>{if (event.alreadyInNcts) 1 else 0}</AlrInNCTCTL29>
-            </CTLCTL>
-            {buildIncident(event.eventDetails)(arrivalNotificationRequest)}
-          </ENROUEVETEV>
+        <PlaTEV10>
+          {event.place}
+        </PlaTEV10> <PlaTEV10LNG>
+        {arrivalNotificationRequest.header.languageCode}
+      </PlaTEV10LNG> <CouTEV13>
+        {event.countryCode}
+      </CouTEV13> <CTLCTL>
+        <AlrInNCTCTL29>
+          {if (event.alreadyInNcts) 1 else 0}
+        </AlrInNCTCTL29>
+      </CTLCTL>{buildIncident(event.eventDetails)(arrivalNotificationRequest)}
+      </ENROUEVETEV>
```

Comment:
Would it be possible to revert this back please, intellij formats makes a mess of xml. Would it be possible to do this in test as well please.